src/expression_evaluator/literal_evaluator.cpp

```diff
@@ -10,6 +10,7 @@ namespace kuzu {
 namespace evaluator {
 
 void LiteralExpressionEvaluator::evaluate() {
+    resultVector->setState(flatState);
```

Comment:
Why is this needed if u already set it in `resolveResultVector` l36?

Reply:
when we populateDefaultVal in storage during add column, we call multiple evaluates but we init only once. For example, if u evaluate 2048, 2048, 1, then the last one will fail since the state remains the 2048 selection size